distributed/utils.py

```diff
@@ -783,7 +784,7 @@ def func(...):
         Set to True to break into the debugger in case of exception
     unroll_stack: int, optional
         Number of levels of stack to unroll when determining the module's name for the
-        purpose of logging. Normally you should omit this. Set to 2 if you are writing a
+        purpose of logging. Normally you should omit this. Set to 1 if you are writing a
```

Comment:
log_errors is not linked anywhere in sphinx, so I believe it should be treated as internal.